src/components/SessionPaymentStatusChip.js

```diff
@@ -1,13 +1,22 @@
 import React from "react";
 import {sessionPaymentStatus} from "../utils";
 
-export const SessionPaymentStatusChip = ({session, enrollment}) => {
+export const SessionPaymentStatusChip = ({session, enrollment, setPos, style}) => {
+    if(!session || !enrollment) {
+        return <div
+            style={style}
+            className={`sessions-left-chip NA ${setPos && "set-pos"}`}>
+            Loading...
+        </div>
+    }
     if(session && enrollment){
         const status = sessionPaymentStatus(session, enrollment);
 
-        return <div className={`sessions-left-chip ${status}`}>
+        return <div
+            style={style}
+            className={`sessions-left-chip ${setPos && "set-pos"} ${status}`}>
             {status}
         </div>
     }
-    return <></>
+    return <div>hi</div>
```

Comment:
Instead of having this you could have just made the component use an `if/else` lol.

Reply:
it. works. :|